src/__tests__/utils/mockBrowserSplitSdk.ts

```diff
@@ -62,7 +62,7 @@ export function mockSdk() {
       });
       const ready: jest.Mock = jest.fn(() => {
         return promiseWrapper(new Promise((res, rej) => {
-          __isReady__ ? res() : __emitter__.on(Event.SDK_READY, res);
+          __isReady__ ? res(null) : __emitter__.on(Event.SDK_READY, res);
```

Comment:
I would change it into
```
        return promiseWrapper(new Promise<void>((res, rej) => {
          __isReady__ ? res() : __emitter__.on(Event.SDK_READY, res);
```
to avoid changing the signature of the ready method.